speaker_doc.py

```diff
@@ -0,0 +1,23 @@
+import json
+
+from app.instance import current_app
+from app.models import db
+
+
+def db_migrate_speaker_doc(db):
+
+    # migrating slides url from url type to JSON type
+
+    conn = db.engine.connect()
+    cursor = conn.execute("SELECT * from sessions")
+
+    for row in cursor:
+        dict = {'name': row['title'], 'link': row['slides_url']}
+        data = f"'{json.dumps(dict)}'"
```

Comment:
```suggestion
    dict = []
    for row in cursor:
        dict.append({'name': row['title'], 'link': row['slides_url']})
        data = f"'{json.dumps(dict)}'"
```

Reply:
Pushing object inside array for allowing multiple objects in future.

Reply:
@MeghalBisht  thanks :D